core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1146,6 +1206,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
    * filesystems), an HTTP, HTTPS or FTP URI, or local:/path for a file on every worker node.
    */
   def addJar(path: String) {
+    assertNotStopped()
```

Comment:
This is another one of those toss-up cases: it works some of the time, so I'll remove this check for conservatism's sake.